lint/attributes.go

```diff
@@ -0,0 +1,13 @@
+package lint
+
+// TODO(quasilyte): move attribute-related things here
+// when this refactoring is complete.
+
+type checkerAttribute int
+
+// All valid attributes.
+var (
+	attrExperimental    = new(checkerAttribute)
```

Comment:
To be honest I will expect flags, not a pointer. So:
```
attrExperimental checkerAttribute =  1 << iota
attrSyntaxOnly
attrVeryOpinionated
```
But it's kinda a minor change. so np